server/index.js

```diff
@@ -157,6 +169,10 @@ module.exports = (_config = {}) => {
       app.use('/bwallet', bcoinRouter(walletClient));
     }
 
+    if (multisigWalletClient) {
+      app.use('/multisig', bcoinRouter(multisigWalletClient));
```

Comment:
do we want to use `bmultisig` as the path to make any collisions from plugins less likely?

Reply:
`/multisig` is hardcoded into the multisig client so we have to use that path

Reply:
these just route to the client on the bpanel server though. The calls from bpanel -> bcoin/bmultisig server can't be different, but the path for calls from bpanel client to bpanel server should be able to be whatever we want right? (Just like wallet calls need to be `/wallet` but we "namespace" it to `bwallet` for bpanel requests. Would need to update the multisig client in bpanel-utils though.

Reply:
`multisigClient.getInfo()` --> `GET /multisig`

This is hardcoded into the multisig client here: https://github.com/bcoin-org/bmultisig/blob/master/lib/client.js#L20

Reply:
Yeah but this route isn't calling the multsig server directly. This is setting up a route for the client to communicate with the bpanel server which proxies the request to the multisig server. You could have the path be anything you want, e.g. `app.use('/asdfasdf/', bcoinRouter(multisigWalletClient))` so when a request from the client comes for `/asdfasdf` it will then route it through the `multisigWalletClient` which does query `/multisig`. This is just like what we do for `bwallet` and `bcoin` above. the wallet requests have to query `/wallet`, but are proxied via the `bwallet` endpoint. 

If we change this, we will have to update this in bpanel-utils though: https://github.com/bpanel-org/bpanel-utils/blob/master/lib/clients.js#L46

Reply:
https://github.com/bcoin-org/bmultisig/blob/master/lib/client.js#L20

Reply:
The path is assigned here - https://github.com/bcoin-org/bcurl/blob/master/lib/client.js#L34
and is then overwritten above, so no matter what you put as the path, it will be the hardcoded `/multisig`

Reply:
Got it! My bad 😅. Submitted an issue: https://github.com/bcoin-org/bmultisig/issues/15